lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyDiagnosticParticipant.java

```diff
@@ -150,11 +154,14 @@ private void validateIncludeLocation(DOMDocument domDocument, List<Diagnostic> d
         if (locAttribute.startsWith("http") || locAttribute.contains("$")) {
             return;
         }
+        // Liberty uses this to determine if directory. 
+        boolean isLibertyDirectory = locAttribute.endsWith("/") || locAttribute.endsWith(File.separator);
 
         DOMNode locNode = node.getAttributeNode("location");
         Range range = XMLPositionUtility.createRange(locNode.getStart(), locNode.getEnd(), domDocument);
-        if (!locAttribute.endsWith(".xml")) {
-            String message = "The specified resource is not an XML file.";
+        if (!locAttribute.endsWith(".xml")
+         && !isLibertyDirectory) {
+            String message = "The specified resource is not an XML file or directory.";
```

Comment:
This message might be more helpful in case they were trying to specify a directory:
`The specified resource is not an XML file. If it is a directory, it must end with a file separator.`

Do you think file separator is good, or should we just say forward slash?

Reply:
I think `file separator` or `trailing slash` is good. 
In Windows, it can be either slash, but we don't check for consistent usage of the same type of slash.